confignode/src/main/java/org/apache/iotdb/confignode/manager/ConsensusManager.java

```diff
@@ -83,6 +84,13 @@ private void setConsensusLayer(PartitionRegionStateMachine stateMachine) throws
                 ConsensusConfig.newBuilder()
                     .setThisNode(new TEndPoint(CONF.getInternalAddress(), CONF.getConsensusPort()))
                     .setStorageDir(CONF.getConsensusDir())
+                    .setRatisConfig(
+                        RatisConfig.newBuilder()
+                            .setSnapshot(
+                                RatisConfig.Snapshot.newBuilder()
+                                    .setAutoTriggerThreshold(100)
```

Comment:
Is this ok to set the `autoTriggerThreshold` config to 100 in master branch？

Reply:
@SzyWilliam has fixed this on [IOTDB-4545](https://github.com/apache/iotdb/pull/7461), I'll merge that PR later~

Reply:
Fixed~